extensions/sql-database-projects/src/dialogs/deployDatabaseQuickpick.ts

```diff
@@ -104,12 +104,27 @@ export async function launchPublishToDockerContainerQuickpick(project: Project):
 		return undefined;
 	}
 
+	let baseImage: string | undefined = '';
+	baseImage = await vscode.window.showInputBox({
+		title: constants.enterBaseImage,
+		ignoreFocusOut: true,
+		value: constants.defaultDockerBaseImage,
+		validateInput: input => utils.isEmptyString(input) ? constants.valueCannotBeEmpty : undefined
```

Comment:
what happens if the image name give isn't a valid image? Are there a set number of images that could be listed as options for the user to choose or are there too many?

Reply:
we don't validate the image right now. building docker container will fail if it's invalid. there's not exactly a list to verify because user can use different tags also (latest)